RxJavaTemplate/app/src/main/java/co/nimblehq/rxjava/ui/base/BaseViewModel.kt

```diff
@@ -12,7 +12,7 @@ import io.reactivex.subjects.PublishSubject
 @Suppress("PropertyName")
 abstract class BaseViewModel : ViewModel() {
 
-    protected val _showLoading = BehaviorSubject.createDefault(false)
+    private val _showLoading = BehaviorSubject.createDefault(false)
     protected val _error = BehaviorSubject.create<Throwable>()
     protected val _navigator = PublishSubject.create<NavigationEvent>()
```

Comment:
@luongvo I think we should group these `private` and `protected` vals with theirs public accessors as per discussed convention from our working projects, what do you think? 🙏

Reply:
@minhnimble okay, now I follow this https://github.com/nimblehq/compass/issues/559

Reply:
I think there are few places in the PR also need grouping like https://github.com/nimblehq/compass/issues/559. You check and clean them as well 🙏
```
private val _showLoading = BehaviorSubject.createDefault(false)
val showLoading: Observable<IsLoading>
        get() = _showLoading
```

Reply:
@minhnimble updated all by this commit https://github.com/nimblehq/android-templates/pull/66/commits/527e9592d5db943c24b947c1f57c150839c8d2ee